torch/distributed/nn/functional.py

```diff
@@ -88,26 +110,60 @@ def all_gather(tensor, group=dist.group.WORLD):
         group (ProcessGroup, optional): The process group to work on.
 
     Returns:
-        tuple[Tensor]): Output of the collective.
+        tuple([Tensor]): Output of the collective.
 
     """
     return _AllGather.apply(group, tensor)
 
 
-def all_to_all(tensors, group=dist.group.WORLD):
+def all_to_all(output_tensor_list, input_tensor_list, group=dist.group.WORLD):
     """
     Each process scatters list of input tensors to all processes in a group and
     return gathered list of tensors in output list.
 
     Arguments:
         tensors (list[Tensor]): List of tensors to scatter one per rank.
         group (ProcessGroup, optional): The process group to work on.
+        out_tensor_list (list[Tensor]): list of tensors to gather one per rank.
```

Comment:
The args in the docstring do not seem to match the ones in the code?

docstring: tensors, group, out_tensor_list
code: output_tensor_list, input_tensor_list, group